src/services/article.service.js

```diff
@@ -170,63 +170,32 @@ export default {
 
   getPendingArticles(editorId) {
     return db('articles')
-      .leftJoin('categories', 'articles.category_id', 'categories.category_id')
-      .leftJoin('users', 'articles.editor_id', 'users.user_id')
-      .where('articles.status', 'waiting')
-      .andWhere('users.user_id', editorId)
-      .andWhere(function () {
-        this.where('articles.category_id', function () {
-          this.select('managed_category_id')
-            .from('users')
-            .where('user_id', editorId);
-        }).orWhere('categories.belong_to', function () {
-          this.select('managed_category_id')
-            .from('users')
-            .where('user_id', editorId);
-        });
-      })
-      .select(
-        'articles.article_id',
-        'articles.title',
-        'articles.abstract',
-        'articles.thumbnail',
-        'articles.views',
-        'articles.status',
-        'articles.published_date',
-        'articles.is_premium'
-      )
-      .orderBy('articles.published_date', 'desc');
+    .leftJoin('categories', 'articles.category_id', 'categories.category_id')
```

Comment:
This does not look like the query i wrote yesterday, is this query better?

Reply:
I have refactored again in later commits

Reply:
which commit is it?